acm/src/main/java/org/literacybridge/acm/config/DBConfiguration.java

```diff
@@ -272,6 +273,9 @@ public boolean shouldPreCacheWav() {
                 }
```

Comment:
(This applies to "if (audioLanguages.isEmpty())..." above.)

Maybe comment why we arbitrarily add English?